### ISSUE TITLE
plugin is not loaded in fedora 31 x86_64 after installing lightspark(firefox 76.0.1)

### ISSUE THREAD

**Issue by xiaotangtj127** [lightspark/lightspark#502]
### What is the purpose of your issue?
- [ ] Site Support Request: Lightspark does not work correctly on a specific website.
- [ ] Bug: Lightspark has the same problem across multiple websites or SWF files.
- [ ] Feature request: a request for new features or support for new platforms, *not sites or SWF files*.
- [ ] Question, Comment, or Suggestion
- [x] If none of the above apply, check this box.

---

### The following sections should be filled in based on the purpose listed above. Erase any section (the contents between triple ---) that does not apply.

--- If this issue is a *bug report* or *site support request*:

#### Provide the Lightspark version

Output of ``lightspark -v``:

```
INFO: Lightspark version 0.8.2 Copyright 2009-2013 Alessandro Pignotti and others
```

#### What operating system are you are running on?

- [ ] Windows 7, 8 or 10
- [ ] Windows XP, Vista or older
- [x ] Linux: *<fedora 31>*
- [ ] Mac OS X: *<specify version here, such as 10.8.1>*
- [ ] Other: *<specify OS and version here>*

#### How are you are running Lightspark?

- [ ] Browser plugin for Chrome or Chromium: *<specify Chromium version here>*
- [ ] Browser plugin for Waterfox: *<specify Waterfox version here>*
- [ x] Browser plugin for other browser: *<firefox 76.0.1>*
- [ ] Lightspark executable on the command line

Identify the subject of this issue:

- [ x ] The documentation of how to use Lightspark, including the wiki
- [ x ] The build instructions to compile Lightspark from source
- [ ] The project and how it is managed (people, process, and so on)
- [ ] Other subject matter not listed

---

### Description of your issue

I download source code of version 0.8.2 (tar.gz)  on fedora 31, install depend pkgs and run ./build , everything goes well, then I open firefox about:plugin page, lightspark plugin is not loaded

**Comment by dbluelle** [comment]
If you use the build script, the plugin is installed under /usr/local/lib/mozilla/plugins, I don't know if Firefox looks there for plugins.

**Comment by xiaotangtj127** [comment]
Yes, it is installed in this dir successfully, but i cannot find it at about:plugin page



发自我的iPhone


------------------ Original ------------------

**Comment by dbluelle** [comment]
In that case you can set the environment variable MOZ_PLUGIN_PATH to /usr/local/lib/mozilla/plugins and then Firefox should find it.

**Comment by xavierba** [comment]
Just for the record, rpm packages for Fedora are available from RPM Fusion.
Although, I usually provide packages only for releases, less frequently from in-between snapshots, so as lightspark 0.8.2 is getting old, there are probably quite some missing features compared to master.

**Comment by dbluelle** [comment]
Yeah, I'm planning to release a new version quite soon...

**Comment by ousia** [comment]
@xiaotangtj127,

could you confirm that the issue is still valid?

Maybe it has been fixed with the release of version 0.8.3 and its package for Fedora by @xavierba.

Cheers.
